### PR TITLE
Added trait for ContainerAwareInterface

### PR DESCRIPTION
**Pull request by hannesvdvreken**
(duplicate of #38 but to league branch)

### CODE REVIEW

File: src/ContainerAwareTrait.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+trait ContainerAwareTrait
+{
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
+
+    /**
+     * Set a container
```
Comment (GrahamCampbell) [inline src/ContainerAwareTrait.php]: missing fullstop

File: src/ContainerAwareTrait.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+     *
+     * @param \Orno\Di\ContainerInterface $container
+     */
+    public function setContainer(ContainerInterface $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * Get the container
```
Comment (GrahamCampbell) [inline src/ContainerAwareTrait.php]: missing fullstop

File: src/ContainerAwareTrait.php (hunk 1 above)
Comment (hannesvdvreken) [inline src/ContainerAwareTrait.php]: Lemme make a new PR for that to fix every single docblock.

**Comment by philipobenito** [comment]
#38

**Pull request closed without merging**